lib/src/select/Select.tsx

```diff
@@ -289,17 +291,10 @@ const DxcSelect = React.forwardRef<RefType, SelectPropsType>(
       [handleSelectChangeValue, closeOptions, multiple]
     );
 
-    const handleListboxResize = () => {
+    const getSelectWidth = useCallback(() => {
       const rect = selectRef?.current?.getBoundingClientRect();
-      setListboxStyles({ width: rect?.width });
-    };
-    useEffect(() => {
```

Comment:
remove useEffect import since it is no longer used.

Reply:
Done!